criu/cr-restore.c

```diff
@@ -3201,6 +3201,35 @@ static void rst_reloc_creds(struct thread_restore_args *thread_args,
 	thread_args->creds_args = args;
 }
 
+static bool groups_match(gid_t* groups, int n_groups)
+{
+	int n, len;
+	bool ret;
+	gid_t* gids;
+
+	n = getgroups(0, NULL);
```

Comment:
pls report an error if getgroups failed

Reply:
Done! Feel free to let me know if you'd like the error message to be modified.